install_template/config.yaml

```diff
@@ -1,4 +1,33 @@
 products:
+  - name: EDB Data Migration Service Reader
+    platforms:
+      - name: RHEL 8 or OL 8
+        arch: x86_64
+        supported versions: [2.0]
```

Comment:
@dwicinas  is there any chance that we can get rid of this line for all platforms? The problem is that we have the 2.14.10 version for now, but that might get bumped regularly (without any changes to the installation path). 

Alternatively, if we do need a line for the version, could we do `2.x` ? That way the reference won't get outdated as quickly.

Reply:
PS: applies to all platforms below.

Reply:
@gvasquezvargas We use this approach for a few products that have frequent upgrades that don't require new documentation.

Reply:
Synced with David, and we'll change to 2.